piwheels/initdb/sql/create_piwheels.sql

```diff
@@ -1062,6 +1142,27 @@ $sql$;
 REVOKE ALL ON FUNCTION test_package(TEXT) FROM PUBLIC;
 GRANT EXECUTE ON FUNCTION test_package(TEXT) TO {username};
 
+-- test_package_deleted(package)
```

Comment:
Would prefer a better name for this. The current name (read on its own) could be interpreted as "test the package has been deleted" or "attempt to delete the package and test whether that succeeded". Might be better as `package_marked_deleted`?